app/app/Http/Controllers/VotingListsController.php

```diff
@@ -122,6 +122,40 @@ public function csv(VotingList $votingList)
         return response($csv, 200, ['Content-Type' => 'text/csv']);
     }
 
+    public function json(VotingList $votingList)
+    {
+        $members = $this->members($votingList)->map(function ($member) {
+            return [
+                'member_id' => $member->id,
+                'last_name' => $member->last_name,
+                'first_name' => $member->first_name,
+                'group' => [                
+                    'abbreviation' => $member->group->abbreviation,
+                    'name' => $member->group->name,
+                ],
+                'country' => [
+                    'code' => $member->country->value,
+                    'name' => $member->country->label,
+                ],
+                'position' => $member->pivot->position->label,
+                ];
+        })->toArray();
+
+        $voting= [
+            'title' => $votingList->display_title,
+            'date' => $votingList->date,
+            'result' => $votingList->result->label,
+            'members' => $members,
+        ];
+
+
+        ob_start();
```

Comment:
```suggestion
```

This is not necessary in this case. `ob_start` caches `stdout` and is used in the `csv` endpoint as a workaround, as the `SimpleExcelWriter` only provides methods to write to a file. However, such a workaround is not necessary in this case.